### DISCUSSION TITLE
Identical monitor but only one supports DDC on MBP M1

### DISCUSSION THREAD

**Discussion by feedthejim** [MonitorControl/MonitorControl#1023] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Hi,

I'm having this weird issue with MonitorControl/OSX and I'm not sure who's the culprit. Posting here because I suppose you are the experts.

I have two identical LG monitors that support DDC that are connected to my MBP M1 through a Caldigit dock with DP cables. For some reason, DDC control with MonitorControl only works with one of the DP outputs.

The obvious culprit here would be that the dock has an issue with routing DDC controls to the second DP port but I tried connecting the second monitor with an HDMI cable to the MBP directly and DDC controls didn't work either.

One other thing I noticed though is that I can also only output sound to one of the monitors but not both, which would point toward the OS having an issue with this scenario.

What do you folks think about this ?

### Expected behavior

DDC controls works on both monitors

### Anything else?

<img width="869" alt="image" src="https://user-images.githubusercontent.com/11064311/166763122-e107f03e-f40a-41f8-ad39-e9377b088d04.png">
I ended up disabling DDC for the second monitor so that I could at least use the software controlled dimming.

### Environment Information (please complete the following information)

```markdown
- macOS version: latest
- Mac model: MBP M1 2021
- MonitorControl version: 4.0.2
- Monitor(s): LG 27UN83A
- Monitor Cable(s)/Connection(s): DP through Caldigit usb c dock
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Hi there - No macs since 2018 do support DDC control over HDMI unfortunately - see this: https://github.com/MonitorControl/MonitorControl/discussions/750
